OSDOCS-19394 needed to replace resume create-only information (#110742)
openshift/openshift-docs: modules/zero-trust-manager-restart-reconciliation.adoc

----
$ oc -n $OPERATOR_NAMESPACE patch subscription openshift-zero-trust-workload-identity-manager --type='merge' -p '{"spec":{"config":{"env":[{"name":"CREATE_ONLY_MODE","value":"false"}]}}}'
----

.Verification

> * Check the status of the `SpireServer` resource to confirm that `create-only` mode is disabled. The `CreateOnlyMode` condition must show `status: "False"`.

Reviewer: ```suggestion
* Check the status of the `SpireServer` resource to confirm that `create-only` mode is disabled by running the following command: 
```

Author: added your edits and deleted the note since you are right that it is in the output.
